canonical-web-and-design/jaas-dashboard
# Explicity add modelName as route param

## Done

This is explicity add `userName` and `modelName` as route params on `/models/*`

This refactor is necessary to support application details pages at `/models/:userName/:modelName/:applicationName`

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check code
- Ensure all tests and checks pass
